rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :person, :date
+  attr_reader :book
+
+  def initialize(date, person, book)
+    @date = date
+
+    @person = person
+    person.rentals << self
+
+    @book = book
+    book.rentals << self
```

Comment:
- [ ] Kindly make sure the `Student` can be able to rent a book. as you may notice the error when trying to rent a book. 
![image](https://user-images.githubusercontent.com/97100091/224361294-ed68e61e-6844-463c-a8ca-89af7038978d.png)
Hint:
Why not only say
```rb
@person.rentals << self
@book.rentals << self
```